### ISSUE TITLE
newline symbol in strings in stored procs replaced with line breaks on edit

### ISSUE THREAD

**Issue by rutkas** [phpmyadmin/phpmyadmin#12982]
### Steps to reproduce
1. Create stored proc or trigger with '\n' inside of any string for example:
SET NEW.field = 'hello\nworld'
2. save stored proc
3. open stored proc for editing again

### Expected behaviour
SET NEW.field = 'hello\nworld'
should be left

### Actual behaviour
SET NEW.field = 'hello
world'

"\n" is replaced with line break in the proc editor

### Server configuration
**Operating system**:
Linux 3.6.11-gentoo #1 SMP Thu Jan 31 18:00:09 EET 2013 x86_64

**Web server:**
Server version: Apache/2.4.25 (Unix)

**Database:**
Client API version	mysqlnd 5.0.11-dev - 20120503
mysqld  Ver 5.6.35 for Linux on x86_64 (Source distribution)

**PHP version:**
PHP/5.6.30-pl0-gentoo

**phpMyAdmin version:**
4.6.6

### Client configuration
**Browser:**
Chrome

**Operating system:**
Win 10 Home

**Comment by nijel** [comment]
It works fine in my tests, most likely it's MySQL server bug that it reports corrupt data, see https://github.com/phpmyadmin/phpmyadmin/issues/12396 and https://bugs.mysql.com/bug.php?id=58342.

Can you try getting the trigger body directly from MySQL using following query?

```mysql
SELECT ACTION_STATEMENT FROM `INFORMATION_SCHEMA`.`TRIGGERS` WHERE TRIGGER_NAME="name of your trigger"
```

**Comment by rutkas** [comment]
Yep.. You are right, looks like mysql bug:

mysql> SELECT ACTION_STATEMENT FROM `INFORMATION_SCHEMA`.`TRIGGERS` WHERE TRIGGER_NAME="testTrig"
    ->
    -> ;
+---------------------------------------------------------------------+
| ACTION_STATEMENT                                                    |
+---------------------------------------------------------------------+
| BEGIN
DECLARE str varchar(255);

SET str = "hello
world";

END |
+---------------------------------------------------------------------+
1 row in set (0.09 sec)

**Comment by nijel** [comment]
It's fixed in MariaDB 10.1.12 (see https://jira.mariadb.org/browse/MDEV-7231) if you're looking for fix, but AFAIK still not fixed in MySQL....
